vendor/bat-native-ledger/src/bat/ledger/internal/request/request_util.cc

```diff
@@ -49,15 +93,15 @@ std::string BuildPublisherUrl() {
 
 std::string BuildLedgerUrl() {
   std::string url;
-  switch (ledger::_environment) {
-    case ledger::Environment::STAGING:
-      url = LEDGER_STAGING_SERVER;
+  switch (::ledger::_environment) {
+    case ::ledger::Environment::DEVELOPMENT:
+      url = ledger::kDevelopment;
```

Comment:
It's a bit unfortunate to have two `ledger` namespaces at different levels, particularly when it's the root namespace.

Do we need namespaces here? We could just have, e.g. `kLedgerStaging` for now, and then when we transition to "endpoint" abstractions we can introduce namespaces then.

Reply:
good catch, will adjust

Reply:
done

Reply:
I was thinking of just removing all the namespaces for now and simply having `kLedgerStaging`, `kCdnStaging`, etc. But this seems fine too.